plugins/email_service/app/javascript/email_service.coffee

```diff
@@ -48,11 +48,11 @@ labelSource = 'label[for="templated_email_source"]'
 labelTemplateData = 'label[for="templated_email_template_data"]'
 
 # Test data
-source = "development.solution03@gmail.com"
-replyTo = "development.solution03@gmail.com"
-toAddresses = "sirajudheenam@gmail.com, buzzmesam@gmail.com" 
-ccAddresses = "sirajudheenam@gmail.com, buzzmesam@gmail.com"
-bccAddresses = "sirajudheenam@gmail.com, buzzmesam@gmail.com"
+source = "fake3@gmail.com"
+replyTo = "fake3@gmail.com"
```

Comment:
It would probably make sence here also to use a fake domain. We don't want to advertise gmail.